IANotificationsTestProject/In App Notifications/IANotification.swift

```diff
@@ -0,0 +1,89 @@
+import UIKit
+
+//IANotification is where the content of an in app notification is stored
+//This class contains the window that displays the notification and all of the UI elements
+//This class handles triggering actions if present and tapped
+
+class IANotification: UIViewController {
+    @IBOutlet weak var notificationView: UIStackView!
+    @IBOutlet weak var messageLabel: UILabel!
+    @IBOutlet weak var actionButton: UIButton!
+    var message: String
+    var action: IANAction?
+    var direction: IANDirection
+    var window: UIWindow!
+    
+    var hasAction: Bool {
+        return action != nil
+    }
+    
+    
+    init(message: String, action: IANAction?, direction: IANDirection = .upFromBottom) {
```

Comment:
If this will always be `upFromBotton` then perhaps we don't really need `IANDirection` ?

Reply:
My thought here is that there may be a use for the notifications to be down from top in the future, say if we added a tap to return to the top of a note feature, you could use these in app notifications to facilitate a lot of that functionality really easily, but I would think it would be better for those notifications to come from the top.  

That being said we don't need to have that in this first iteration and I can remove that.  Was trying to future proof the API